modules/interactive_music/audio_stream_interactive.cpp

```diff
@@ -976,6 +976,10 @@ void AudioStreamPlaybackInteractive::switch_to_clip_by_name(const StringName &p_
 		return;
 	}
 
+	if (stream.is_null()) {
```

Comment:
Could make this an error if desired

Reply:
I guess its better to make it an error, since the functions i meant to be used after it starts.

Reply:
Error could be something like "Attempted to switch while not playing back any stream" or something like that.

Reply:
Rest of the PR is fine, great work!

Reply:
Then I'll change this to an error and squash